modules/local/mtx_to_seurat.nf

```diff
@@ -19,10 +19,10 @@ process MTX_TO_SEURAT {
 
     script:
     def aligner = params.aligner
-    if (params.aligner == "cellranger") {
-        matrix   = "matrix.mtx.gz"
-        barcodes = "barcodes.tsv.gz"
-        features = "features.tsv.gz"
+    if (params.aligner == "cellranger" || params.aligner == "cellrangerarc") {
```

Comment:
`if (params.aligner in [ 'cellranger', 'cellrangerarc' ])` maybe can be easier to read and less to write :)